pom.xml

```diff
@@ -23,154 +23,176 @@
-                    <strictCheck>true</strictCheck>
-                    <encoding>${project.build.sourceEncoding}</encoding>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>check-headers</id>
-                        <phase>verify</phase>
-                        <goals>
-                            <goal>check</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>com.code54.mojo</groupId>
-                <artifactId>buildversion-plugin</artifactId>
-                <version>1.0.3</version>
-                <executions>
-                    <execution>
-                        <phase>validate</phase>
-                        <goals>
-                            <goal>set-properties</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
-                <version>3.1.2</version>
-                <executions>
-                    <execution>
-                        <id>analyze</id>
-                        <phase>verify</phase>
-                        <goals>
-                            <goal>analyze-only</goal>
-                        </goals>
-                        <configuration>
-                            <failOnWarning>true</failOnWarning>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-gpg-plugin</artifactId>
-                <version>3.0.1</version>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-release-plugin</artifactId>
-                <version>3.0.0-M5</version>
-            </plugin>
-        </plugins>
-    </build>
-
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.evolvedbinary.lucene.analyzer</groupId>
+	<artifactId>PLMultiAnalyzer</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<description>A Lucene custom analyzer that allow for indexing multiple tokens for a single term</description>
+	<url>https://github.com/evolvedbinary/PLMultiAnalyzer</url>
+	<inceptionYear>2021</inceptionYear>
+	<organization>
+		<name>Evolved Binary</name>
+	</organization>
+	<properties>
+		<maven.compiler.source>8</maven.compiler.source>
+		<maven.compiler.target>8</maven.compiler.target>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<lucene.version>4.10.4</lucene.version>
+		<git.account>evolvedbinary</git.account>
+		<git.base>github.com/${git.account}/${project.artifactId}</git.base>
```

Comment:
As per the eXist-db community call - I think we both agree that using properties for the SCM section makes it hard to understand as a human - can you just repeat the full and correct urls in the scm section and remove these `git.` properties please?